src/TextUI/Output/Printer/DefaultPrinter.php

```diff
@@ -70,8 +70,10 @@ public static function standardError(): self
     private function __construct(string $out)
     {
         $this->isPhpStream = str_starts_with($out, 'php://');
+        $x = 1;
```

Comment:
I assume you remove this commit before we merged this PR ;)

Reply:
Yes. Since the mutation test only works on the PR diff, I need this changes to see if it works :-)